types/config.d.ts

```diff
@@ -5,7 +5,7 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import Audit = require('../lighthouse-core/audits/audit.js');
+import AuditClass = require('../lighthouse-core/audits/audit.js');
```

Comment:
this is what I was talking about in #12860, where some changes lead to `Audit` in this file to refer to the global `LH.Audit` _namespace_, not this very explicit local import to a (type) variable named `Audit`.

This can be reverted when `config.d.ts` moves off global declarations